app/src/app/views/stix/matrix/matrix-list/matrix-list.component.html

```diff
@@ -1,6 +1,21 @@
 <div class="list-page">
     <div class="list-page-header">
-        <h1>matrices</h1>
+        <h1 class="with-help-icon">
+            matrices <mat-icon [popover]="PopoverTag" class="help-icon" aria-label="help">help_outline</mat-icon>
+        </h1>
+
+        <popover-content #PopoverTag 
+                popoverPlacement="bottom"
+                placement="bottom"
+                [animation]="true" 
+                [closeOnClickOutside]="true"
+                 >
+            <div class="popover-description">
+                <p>
+                    The relationship between tactics, techniques, and sub-techniques can be visualized in matrices   
+                </p>
```

Comment:
Given that we don't currently support matrix visualization (until #20 at least) this probably needs to be re-written.

> Matrices are an organized visualization depicting the relationships between tactics, techniques, and sub-techniques. ATT&CK Workbench does not support matrix visualizations at present, but you can still create and edit them for visualization in other tools.